vernac/vernacentries.ml

```diff
@@ -748,13 +748,8 @@ let polymorphic_cumulative ~is_defclass =
   let open Notations in
   (* EJGA: this seems redudant with code in attributes.ml *)
```

Comment:
Is this comment still relevant? cc @ejgallego

Reply:
Well, that still looks somewhat redundant to me, the deprecation/removal of old boolean attributes doesn't change that point IMHO.